meson.build

```diff
@@ -281,7 +281,7 @@ if get_option('default_library') != 'static'
     platform_deps += cc.find_library('shlwapi')
   endif
   if host_machine.system() == 'freebsd'
-    platform_deps += cc.find_library('efivar')
+    platform_deps += dependency('efivar')
   endif
```

Comment:
This is the top-level `meson.build`. `cc.find_library` found base-system library (in `/usr/lib`), which isn't complete, instead of discovering a port of `libefivar` (in `/usr/local/lib`).

Reply:
So just fix the top level one to find the port?

Reply:
This **is** the top-level one :)

Reply:
> This **is** the top-level one :)

🤦sorry about that!